packages/amplify-cli/src/commands/env/get.ts

```diff
@@ -1,40 +1,33 @@
-import chalk from 'chalk';
-import { JSONUtilities, $TSContext, UnknownArgumentError, exitOnNextTick } from 'amplify-cli-core';
+import { JSONUtilities, $TSContext, AmplifyError } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
 import { printEnvInfo } from '../helpers/envUtils';
 
-export const run = async (context: $TSContext) => {
+/**
+ * Executes the 'env get' command
+ */
+export const run = async (context: $TSContext) : Promise<void> => {
   const envName = context.parameters.options.name;
+  const allEnvs = context.amplify.getEnvDetails();
 
   if (!envName) {
-    const errMessage = 'You must pass in the name of the environment using the --name flag';
-    context.print.error(errMessage);
-    context.usageData.emitError(new UnknownArgumentError(errMessage));
-    exitOnNextTick(1);
+    throw new AmplifyError('EnvironmentNameError', {
+      message: 'Environment name was not specified.',
+      resolution: 'Pass in the name of the environment using the --name flag.',
+    });
+  }
+  if (!allEnvs[envName]) {
+    throw new AmplifyError('EnvironmentNameError', {
```

Comment:
same as above

Reply:
also this didn't used to be an error here, are there side-effects to this change?

Reply:
Before, this would fail silently if a user wants to 'get' an invalid environment, user would not get resolutions steps; but now they do